### DISCUSSION TITLE
Where to submit task 2

### DISCUSSION THREAD

**Discussion by shouldersLL** [FETS-AI/Challenge#119] [Q&A]
Dear organizers,

Where should we submit the container for task 2, using the link attached below or using Gitlab as indicated in the readme file of task 2?

Many thanks.

<img width="838" alt="image" src="https://user-images.githubusercontent.com/65354349/126875034-ccc824b8-028c-4668-b792-3649d05e0f20.png">

**Comment by sarthakpati** [comment]
Please see https://fets-ai.github.io/Challenge/participate/#task-2-submission for details on Task-2 submission.
